__tests__/app.test.js

```diff
@@ -43,4 +43,39 @@ describe("GET /api", ()=> {
       expect(body).toEqual(apiEndpoints)
     })
   })
+})
+
+describe('GET /api/articles/:article_id/comments', () => {
+  test('200 accepts an article_id and responds with an array of comments for that given id ', () => {
+    return request(app)
+    .get("/api/articles/1/comments")
+    .expect(200)
+    .then(({body}) => {
+      const {comments} = body;
+      comments.forEach((comment) => {
+        expect(comment).toHaveProperty("comment_id", expect.any(Number));
+        expect(comment).toHaveProperty("votes", expect.any(Number));
+        expect(comment).toHaveProperty("created_at", expect.any(String));
+        expect(comment).toHaveProperty("author", expect.any(String));
+        expect(comment).toHaveProperty("body", expect.any(String));
+        expect(comment).toHaveProperty("article_id", expect.any(Number));
+      })
+    })
+  })
```

Comment:
You're missing a test for a valid ID with no comments. What status would this be? You would also want this to respond with an empty array 👍